README.md

```diff
@@ -42,9 +42,9 @@ Search for `cats` on the engine `my-engine` with filters and facets:
       console.log(res)
     })
 
-### Autocomplete
+### Suggest
```

Comment:
Thanks for the PR! We'd like to keep this as **Autocomplete**. The reason is that Autocomplete is consistent with the [documentation](**https://swiftype.com/documentation/site-search/autocomplete**), through from the product and out to marketing. 👍

Reply:
No problems - I'll switch this back.